src/components/Content/Pursuance/views/DiscussView.js

```diff
@@ -75,6 +75,14 @@ class DiscussView extends Component {
                   }}} />
               </span>
             </div>
+            <div className="subtasks-ctn">
+              {subtaskGids.length > 0 && <h4>Subtasks</h4>}
+            <ul className="subtasks-list">
+            {subtaskGids.map((gid, ind)=> {
+              return <li key={ind} className="subtask-item"><FaCircleO size={10} className="fa-circle-o" />{tasks.taskMap[gid].title}</li>
```

Comment:
@Moe-Shoman How about we always call index variables `i` or `ndx`?

Reply:
@elimisteve sounds good